src/test/java/com/programmers/springbootbasic/domain/customer/JdbcCustomerRepositoryTest.java

```diff
@@ -0,0 +1,43 @@
+package com.programmers.springbootbasic.domain.customer;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.context.annotation.Import;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@JdbcTest
+@Import(JdbcCustomerRepository.class)
+class JdbcCustomerRepositoryTest {
+    @Autowired
+    private JdbcCustomerRepository repository;
+
+    @Test
+    @DisplayName("고객을 저장 할 수 있다")
+    void save() {
+        // given
+        Customer customer = new Customer(UUID.randomUUID(), "test@gmail.com", "test");
+        // when
+        Optional<Customer> save = repository.save(customer);
```

Comment:
```suggestion
        Optional<Customer> result = repository.save(customer);
```
의미있는 이름을 지어주는 것이 좋겠네요